houston/app.go

```diff
@@ -68,12 +68,26 @@ var (
 		},
 	}
 
-	AvailableNamespacesGetRequest = `
+	AvailableNamespacesGetRequest = queryList{
+		{
+			version: "0.25.0",
+			query: `
 	query availableNamespaces {
-		availableNamespaces{
+		availableNamespaces {
 			name
 		}
-	}`
+	}`,
+		},
+		{
+			version: "1.0.0",
+			query: `
+	query availableNamespaces($clusterId: Uuid!) {
+		availableNamespaces(clusterId: $clusterId) {
+			name
+		}
+	}`,
```

Comment:
we have to update our api as well in order to make it work insync with dataplanes

Reply:
It's unrelated to this PR and we can discuss it seperately.